bigquery/google/cloud/bigquery/schema.py

```diff
@@ -17,6 +17,8 @@
 from google.cloud.bigquery_v2 import types
 
 
+STRUCT_TYPES = ("RECORD", "STRUCT")
```

Comment:
I think we should make this "private" for now (leading underscore), as I'm having trouble thinking of a customer use-case for these constants.

Reply:
`_pandas_helpers.py` make use of this as well, thus I would personally keep this "public", as `schema.STRUCT_TYPE` can serve other modules (even if those modules happen to be private).

If you have a strong opinion about it, I can still change it, just putting this up for consideration.

Reply:
Other modules can use it just fine, even if it's "private". Any breaking changes in that case would be caught by our test suite.

Reply:
We should always err on the side of not introducing new public APIs, since keeping things private gives us more flexibility.

Reply:
Makes sense, will change.